fdbserver/Ratekeeper.actor.cpp

```diff
@@ -251,6 +251,9 @@ class RatekeeperImpl {
 		state Transaction tr(db);
 		loop {
 			try {
+				tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
+				tr.setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
+				tr.setOption(FDBTransactionOptions::LOCK_AWARE);
```

Comment:
I could see how this transaction could hang if `LOCK_AWARE` isn't on, since if it was run on a locked database it would get a retryable error. Is this actually accessing system keys, though? If that were failing, it should throw an error.

Reply:
Ah, the reason we don't need `READ_SYSTEM_KEYS` here because this is a NativeAPI transaction